hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/HapiApiClients.java

```diff
@@ -119,14 +149,53 @@ private void addStubs(NodeConnectInfo node, String uri, boolean useTls) {
         }
     }
 
-    private HapiApiClients(List<NodeConnectInfo> nodes, AccountID defaultNode) {
+    /**
+     * Since we need to submit workflow operations to a different port, we need to create a channel for that port.
+     * @param node Nodes to connect to
+     * @param uri String specifying host, port
+     * @param useTls true if tls is used, false otherwise
+     * @param workflowOperations set of HAPI operations client should submit to a different port
+     */
+    private void addNewNettyChannelForWorkflowOperations(
+            NodeConnectInfo node, String uri, boolean useTls, Set<HederaFunctionality> workflowOperations) {
+        Set<HederaFunctionality> consensusOps = Set.of(
+                ConsensusCreateTopic,
+                ConsensusDeleteTopic,
+                ConsensusGetTopicInfo,
+                ConsensusSubmitMessage,
+                ConsensusUpdateTopic);
+        String workflowUri = "";
+        if (uri.equals(node.uri())) {
+            workflowUri = node.workflowUri();
+        } else if (uri.equals(node.tlsUri())) {
+            workflowUri = node.workflowTlsUri();
+        }
+
+        ManagedChannel workflowChannel =
+                createNettyChannel(useTls, node.getHost(), node.getWorkflowPort(), node.getWorkflowTlsPort());
+        channels.put(workflowUri, workflowChannel);
+
+        log.info("New URI for workflows {}", workflowUri);
+
+        if (workflowOperations.stream().anyMatch(consensusOps::contains)) {
```

Comment:
And we can delete this too; we know that we're only supporting HCS operations right now, so we unconditionally create the HCS stub,
```
consSvcStubs.put(workflowUri, ConsensusServiceGrpc.newBlockingStub(workflowChannel));
```
(As we support other service types, we will just add them here too.)